release-1.8/release_notes_draft.md

```diff
@@ -37,19 +37,21 @@ please check out the [release notes guidance][] issue.
 
 ## **Major Themes**
 
-- The kubernetes workloads API (the DaemonSet, Deployment, ReplicaSet, and
-StatefulSet kinds) have been moved to the new apps/v1beta2 group version. This
-is the current version of the API, and the version we intend to promote to
-GA in future releases. This version of the API introduces several deprecations
-and behavioral changes, but its intention is to provide a stable, consistent
-API surface for promotion.
-
 - The roles based access control (RBAC) API group for managing API authorization
 has been promoted to v1. No changes were made to the API from v1beta1. This
 promotion indicates RBAC's production readiness and adoption. Today, the
 authorizer is turned on by default by many distributions of Kubernetes, and is a
 fundamental aspect of a secure cluster.
 
+### SIG Apps
+
+[SIG Apps][] is responsible for deploying and operating applications in Kubernetes, and focuses on the developer and devops experience of running applications.
+
+For the 1.8 release, SIG Apps moved the kubernetes workloads API, which includes the DaemonSet, Deployment, ReplicaSet, and StatefulSet kinds, to the new apps/v1beta2 group version. SIG Apps intends to promote this version to GA in a future release. Though apps/v1beta2 group version introduces several deprecations and behavioral changes, it provides a stable, consistent API surface for promotion.
```

Comment:
you may want to break this line up. not into multiple sentences but just to avoid word wrapping. I'm not a stickler for this kind of thing but 80-90 characters seems to display reasonably well on most screens (at least when viewing the raw markdown before rendering)